schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals(
    escape_attempts integer,
    neutered boolean,
    weight_kg decimal,
+   species varchar(50),
```

Comment:
- [ ] Instead of creating the table again with the new column, please alter the table _**outside**_ the CREATE TABLE statement.  

Example:

`ALTER TABLE animals ADD species;`